create-snowpack-app/app-template-preact-typescript/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@snowpack/app-template-preact-typescript",
+  "version": "1.10.2",
```

Comment:
Should it be 1.0.0?

Reply:
We can make it `1.0.0`, yeah. Admittedly the version numbers for the templates have gone through a bit of thrashing with our migration into the Snowpack monorepo, as Lerna bumps them for dependency changes and Snowpack changes.